lib/definitions/android-plugin-migrator.d.ts

```diff
@@ -0,0 +1,12 @@
+
+interface IBuildOptions {
```

Comment:
As these options are specific for Android and for building plugins, I think the name should be more concrete: `IAndroidBuildOptions` or `IBuildAndroidPluginOptions`